javav2/example_code/iam/src/main/java/com/example/iam/CreateUser.java

```diff
@@ -1,4 +1,4 @@
-//snippet-sourcedescription:[CreateUser.java demonstrates how to create an AWS Identity and Access Management (IAM) user by using waiters.]
+//snippet-sourcedescription:[CreateUser.java demonstrates how to create an AWS Identity and Access Management (AWS IAM) user by using waiters.]

```

Comment:
AWS Identity and Access Management (IAM)

Reply:
Fixed